### PR TITLE
Removed SonataAdmin integration

### PR DESCRIPTION
**Pull request by wouterj**
Much simpler testing, almost 1000 lines removed and less dependencies. I'm loving this so much ❤️ 

/requires https://github.com/symfony-cmf/sonata-admin-integration-bundle/pull/8 <s>https://github.com/symfony-cmf/core-bundle/pull/210 </s>

### CODE REVIEW

File: UPGRADE-2.1.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Upgrade from 2.0 to 2.1
 
+## Sonata Admin
+
+* All Sonata Admin integration has been removed. The integration is now
+  available via the CmfSonataAdminIntegrationBundle.
```
Comment (dbu) [inline UPGRADE-2.1.md]: should we say "All Sonata Admin integrations have been moved to the CmfSonataAdminIntegrationBundle."? just so the shock of "has been removed" can be avoided...

**Comment by dbu** [comment]
cool! can you check styleci please?

**Comment by ElectricMaxxx** [comment]
merged some StileCI hints. But i would say lets move that, when a PR to insert into sonata-admin-integration is green, right?

File: composer.json [hunk 2]
```diff
@@ -24,16 +24,11 @@
...
-        "sonata-project/doctrine-phpcr-admin-bundle": "^2.0",
-        "sonata-project/admin-bundle": "^2.3.7|^3.0",
-        "sonata-project/block-bundle": "^2.2.8|^3.0",
-        "sonata-project/core-bundle": "^2.2.5|^3.0"
+        "twig/twig": "^1.18|^2.0"
     },
     "minimum-stability": "dev",
     "prefer-stable": true,
     "suggest": {
-        "sonata-project/doctrine-phpcr-admin-bundle": "if you want to have editing capabilities (^1.1)",
```
Comment (wouterj) [inline composer.json]: To be honest, I don't like suggests that much. I see not much value in suggesting this package, people will just read the docs and see that they can use this integration bundle when they're using Sonata.

**Comment by dbu** [comment]
i don't think it hurts to suggest it. but i have no strong opinion
either way. except that we should be consistent through all bundles and
either mention it nowhere or everywhere.

File: composer.json (hunk 2 above)
Comment (ElectricMaxxx) [inline composer.json]: As two examples (seo, routing) are still merged with that suggestion, i think we should add it here and in the other. Lets have a little voting afterwards.

Even when i think those suggestions doen't really matter (and they are ugly white while an composer run), but from a marketing view, we should use every little chanel to show our ideas - even the really little ones.

Comment (dbu) [inline composer.json]: lets consolidate that at some point. maybe its best to have the suggest in CoreBundle only, as that is always installed anyways.

**Comment by dbu** [comment]
thanks wouter!

**Pull request merged**